src/UDB2/EventRepository.php

```diff
@@ -350,6 +397,10 @@ private function importDependencies(\CultureFeed_Cdb_Item_Event $udb2Event)
 
     public function applyEventCreated(EventCreated $eventCreated, Metadata $metadata)
     {
+
+        // Don't send to UDB2 for now.
+        return $eventCreated->getEventId();
```

Comment:
This can't be merged like this as none of the code that follows is being executed. There's a toggle in this class (syncBack) which should be checked and used for this purpose instead.